tests/e2e_testdata.go

```diff
@@ -0,0 +1,514 @@
+package tests
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/adshmh/meter/db"
+	"github.com/gojektech/heimdall/httpclient"
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	influxAPI "github.com/influxdata/influxdb-client-go/v2/api"
+	influxDomain "github.com/influxdata/influxdb-client-go/v2/domain"
+	"github.com/pokt-foundation/portal-api-go/repository"
+	stringUtils "github.com/pokt-foundation/utils-go/strings"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO move this to utils-go
+func startOfDay(day time.Time) time.Time {
+	y, m, d := day.Date()
+	l := day.Location()
+
+	return time.Date(y, m, d, 0, 0, 0, 0, l)
```

Comment:
From GO's standard `time` package examples, the following may read better:
```
func startOfDay(t time.Time) time.Time {
     return time.Date(t.Year(), t.Month(), t.Day(), 0, 0, 0, 0, t.Location())
}
```

Reply:
Sure NP. I will also move it to the `utils-go` `time` package.